src/onnx/parse_convolution.cpp

```diff
@@ -149,9 +159,45 @@ struct parse_convolution : op_parser<parse_convolution>
 
         recalc_conv_attributes(values, kdims);
 
+        instruction_ref ret;
+        // parse a_zero_point and b_zero_point values
+        auto l0_zp         = set_bias(l0, 2, args);
```

Comment:
Referencing https://onnx.ai/onnx/operators/onnx__ConvInteger.html, we could call l0 as x, or something like that..

Request: please rename l0 as x. I am having a very hard time differentiating between l0 and 10. _I know it is not your code, but it is time to perhaps clean it, as it is making code reviews harder._

Reply:
Done